### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by Damla78**

### CODE REVIEW

File: Week1/sql.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    });
+    console.log("----------------------------------");
+  });
+
+}
+
+
+
+
+connection.end();
```
Comment (lao) [inline Week1/sql.js]: I am skipping checking the files from first week. =)

File: Week2/part1.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+const countryLang2 = [
+  {
+    'countryCode': 'KEN',
+    'language': 'XXX',
+    'isOfficial': 'F',
+    'percentage': 13,
+  }
+];
+
+switch (process.argv[2]) {
```
Comment (lao) [inline Week2/part1.js]: Nice that you are getting the arguments like this. =)

File: Week2/part1.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  case 'help':
+    console.log("Usage of the program:");
```
Comment (lao) [inline Week2/part1.js]: +1

File: Week2/part1.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    console.log("node part1.js help");
+    console.log("node part1.js 1 'TURKEY' (1. What is the capital of country X ? )");
+    console.log("node part1.js 2 'Middle East' (2. List all the languages spoken in the region Y )");
+    console.log("node part1.js 3 'Turkish' (3. Find the number of cities in which language Z is spoken )");
+    console.log("node part1.js 4 'Turkish' (4. Are there any countries that have A) Same official language B) Same region If yes, display those countries. If no, display TRUE or FALSE)");
+    console.log("node part1.js 5 (5. List all the continents with the number of languages spoken in each continent)");
+    console.log("node part1.js 6 'KEN' 'XXX' 'F' 13");
+    break;
+  case '1':
+    getCapitalOfCountry(process.argv[3]);
```
Comment (lao) [inline Week2/part1.js]: In this case, you need to consider the async nature of the query. You might want to refactor your functions and use ```await``` and ```async```. I posted some great examples of that in slack. =)

File: Week2/part2.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world'
+});
+//select * from (select countryCode, count(language) as sumLang from countryLanguage group by countryCode) as a where sumLang>9;
+connection.connect();
+
+function insertNewLang()
```
Comment (lao) [inline Week2/part2.js]: I am getting a syntax error in this line ;)

File: Week2/part1.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    insertNewLang(process.argv[3], process.argv[4], process.argv[5], process.argv[6]);
+    break;
+  default:
+    console.log('Wrong choice');
+}
+
+
+//1. What is the capital of country X ? (Accept X from user)
+function getCapitalOfCountry(name) {
+  const query = 'select * from city where id = (select Capital from country where name =\"' + name + '\")';
```
Comment (zhaobn) [inline Week2/part1.js]: You can use the "escape" functionality provided by mysql.js to protect yourself from malicious user input https://github.com/mysqljs/mysql#escaping-query-values

**Pull request closed without merging**